plugins/patchmanager.koplugin/main.lua

```diff
@@ -0,0 +1,188 @@
+        -- strip anything after ".lua" in patch_name
+        local patch_name = patch
+        patch_name = patch_name:sub(1, patch_name:find(ext, 1, true) + ext:len() - 1)
+        table.insert(sub_menu, {
+            text = patch_name .. getExecutionStatus(patch_name),
+            checked_func = function()
+                return patch:find("%.lua$") ~= nil
+            end,
+            callback = function()
+                local extension_pos = patch:find(ext, 1, true)
+                if extension_pos then
+                    local is_patch_enabled = extension_pos == patch:len() - (ext:len() - 1)
+                    if is_patch_enabled then -- patch name ends with ".lua"
+                        local disabled_name = patch .. self.disable_ext
+                        os.rename(self.patch_dir .. "/" .. patch,
+                                  self.patch_dir .. "/" .. disabled_name)
+                        patch = disabled_name
+                    else -- patch name name contains ".lua"
+                        local enabled_name = patch:sub(1, extension_pos + ext:len() - 1)
+                        os.rename(self.patch_dir .. "/" .. patch,
+                                  self.patch_dir .. "/" .. enabled_name)
+                        patch = enabled_name
+                    end
+                end
+                UIManager:askForRestart(T(
+                    _("Patches changed. %1\n"),
+                    _("Current set of patches will be applied on next restart.")))
+            end,
+            hold_callback = function()
+                local patch_fullpath = self.patch_dir .. "/" .. patch
+                if self.ui.texteditor then
+                    self.ui.texteditor.whenDoneFunc = function()
+                        UIManager:askForRestart(T(
+                            _("Patches might have changed. %1\n"),
+                            _("Current set of patches will be applied on next restart.")))
+                    end
+                    self.ui.texteditor:checkEditFile(patch_fullpath, true)
+                else -- fallback to show only the first lines
+                    local message = ""
+                    for line in io.lines(patch_fullpath) do
+                        local line_start = line:sub(1, 1)
+                        if line_start == " " or line_start == "-" then
+                            message = message .. line .. "\n"
+                        else
+                            break
+                        end
+                    end
+
+                    UIManager:show(InfoMessage:new{
+                            text = message,
+                            show_icon = false,
+                            width = math.floor(Screen:getWidth() * 0.9),
+                        })
+                end
+            end,
+        })
+    end
+    return sub_menu
+end
+
+local about_text = _([[Patch manager allows to enable, disable or edit certain found user provided patches.
```

Comment:
Why "certain found" ? :)
```
Patch manager allows to enable, disable and edit user patches.
(The execution time (I'd like to find another wording for that, step? it's like rc2.d/... on Unix) can not (yet) be modified by Patch manager, and have to be done manually by renaming the patch prefix.)

For more info about user patches, see
https://github.com/koreader/koreader/wiki/User-patches
```

Reply:
Or "runlevel and priority" would be more precise.